app/src/main/java/edu/gatech/cs2340/hkskh/User.java

```diff
@@ -0,0 +1,60 @@
+package edu.gatech.cs2340.hkskh;
+
+/**
+ * Created by Kirby on 2/16/2018.
+ */
+
+public class User {
+    private String name;
+    private UserType type;
+    private String pass;
+
+    /**
+     * Creates a new user
+     *
+     * @param name name of the user
+     * @param type the user type
+     * @param pass the password that is needed to give this user access
+     */
+    public User(String name, UserType type, String pass){
+        this.name = name;
+        this.type = type;
+        this.pass = pass;
+    }
+
+    /**
+     * No-arg constructor. Do not call this directly
+     */
+    public User() {
```

Comment:
Can you delete this constructor? Classes don't need empty constructors do they? Or is this useful for something?

Reply:
Yeah we really don't need it as long as we never call it. I guess I put this in as a safety net because I seem to remember learning that it was good coding practice to always have a default no-arg constructor